jobs/speak-clearly-llc-python-telephony-developer.html

```diff
@@ -0,0 +1,42 @@
+---
+title: Python telephony developer
```

Comment:
"Telephony Developer" would be a more appropriate title, given that all jobs on this site are Python.